.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
+    commit-message:
+      prefix: "[Dependabot] "
+    open-pull-requests-limit: 6
```

Comment:
what happens when we reach the limit? Just ignore?

Reply:
it wont open more PRs
this is just to make sure the dependabot wont open 1000 prs at a time ;) 
so when it reaches 6 prs, and it actually need to fix 7 things, so it the last one will be not be open up until you merge one PR and then it will be opened as well

Reply:
tl;dr - it just a maximum amount of prs could be opened at once

Reply:
If the limit is reached, no further pull requests are raised until some of those open requests are merged or closed.